src/testHelpers/dataMocks.ts

```diff
@@ -0,0 +1,37 @@
+import { faker } from '@faker-js/faker';
+
+import { Participant, ParticipantDTO, ParticipantStatus } from '../api/entities/Participant';
+import { UserJobFunction } from '../api/entities/User';
+import { UserWithIsApprover } from '../api/services/usersService';
+
+export const createMockUser = (participants: Participant[]): UserWithIsApprover => {
+  const firstName = faker.person.firstName();
+  const lastName = faker.person.lastName();
+  const generatedEmail = faker.internet.email({
+    firstName,
+    lastName,
+  });
+
+  const user = {
+    id: faker.number.int(),
+    firstName,
+    lastName,
+    email: generatedEmail,
+    jobFunction: faker.helpers.arrayElement(Object.values(UserJobFunction)),
+    acceptedTerms: true,
+    isApprover: faker.datatype.boolean(),
+    participants,
+  };
+
+  return user;
+};
+
+export const createMockParticipant = (): ParticipantDTO => ({
+  id: faker.number.int(),
+  name: faker.company.name(),
+  status: ParticipantStatus.Approved,
+  allowSharing: true,
+  completedRecommendations: faker.datatype.boolean(),
+  crmAgreementNumber: '12345678',
+  users: [createMockUser([])],
```

Comment:
This seems a little weird to me.  You're creating a random user that's unattached to the current participant.  Elsewhere in the code you create the participant and then creating the user and pass in the participant.

Reply:
true - I've removed the line so that it lines up with the other pattern. It wasn't doing anything anyway.